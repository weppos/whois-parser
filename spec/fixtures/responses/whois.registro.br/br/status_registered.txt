
% Copyright (c) Nic.br
%  The use of the data below is only permitted as described in
%  full by the terms of use at https://registro.br/termo/en.html ,
%  being prohibited its distribution, commercialization or
%  reproduction, in particular, to use it for advertising or
%  any similar purpose.
%  2017-06-29 06:38:34 (BRT -03:00)

domain:      google.com.br
owner:       Google Brasil Internet Ltda
ownerid:     06.990.590/0001-23
responsible: Domain Administrator
country:     BR
owner-c:     DOADM17
admin-c:     DOADM17
tech-c:      DOADM17
billing-c:   NAB51
nserver:     ns1.google.com
nsstat:      20170628 AA
nslastaa:    20170628
nserver:     ns2.google.com
nsstat:      20170628 AA
nslastaa:    20170628
nserver:     ns3.google.com
nsstat:      20170628 NOT SYNC ZONE
nslastaa:    20170628
nserver:     ns4.google.com
nsstat:      20170628 NOT SYNC ZONE
nslastaa:    20170628
created:     19990518 #162310
changed:     20170427
expires:     20180518
status:      published

nic-hdl-br:  DOADM17
person:      Domain Admin
e-mail:      ccops@markmonitor.com
country:     BR
created:     20100520
changed:     20170103

nic-hdl-br:  NAB51
person:      NameAction do Brasil
e-mail:      mail@nameaction.com
country:     BR
created:     20020619
changed:     20170602

% Security and mail abuse issues should also be addressed to
% cert.br, http://www.cert.br/ , respectivelly to cert@cert.br
% and mail-abuse@cert.br
%
% whois.registro.br accepts only direct match queries. Types
% of queries are: domain (.br), registrant (tax ID), ticket,
% provider, contact handle (ID), CIDR block, IP and ASN.
