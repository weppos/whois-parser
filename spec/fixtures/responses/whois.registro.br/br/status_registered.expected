#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{1999-05-18}

#updated_on
  %s %CLASS{time}
  %s %TIME{2017-04-27}

#expires_on
  %s %CLASS{time}
  %s %TIME{2018-05-18}


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.google.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.google.com"
